contracts/modules/Payments.sol

```diff
@@ -41,14 +40,13 @@ abstract contract Payments is PaymentsImmutables {
     /// @param recipient The address that will receive payment
     /// @param bips Portion in bips of whole balance of the contract
     function payPortion(address token, address recipient, uint256 bips) internal {
-        if (bips == 0 || bips > FEE_BIPS_BASE) revert InvalidBips();
```

Comment:
Do you not want to check bips == 0 ?

Reply:
I dont really see that it gains anything